.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on:
+  - pull_request
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: false
+          platforms: linux/amd64,linux/arm64,linux/arm64/v8
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Should fixed tag to dev， because of  shell can not get tag from pr

Reply:
Actually, we don't need such complex statements. How about using `make image` instead? Image tag is not important. This step just makes sure the image can build successfully.

Reply:
Absolutely you are right, please change it,thx